src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -96,7 +96,7 @@
 				</div>
 			</dl>
 
-			<h3>{{ t('spreed', 'Shortcuts while in a call') }}</h3>
+			<em>{{ t('spreed', 'Shortcuts while in a call.') }}</em>
```

Comment:
I prefer h3 here. with the light grey it becomes less readable and feels then more like noise opposed to helping orientate in the long list.

Reply:
for accessibility, if you remove h3 you'll need to add `role='heading' aria-level='3'` to keep the heading semantics for screen readers and others

in general I think it's a bad idea to rely on html markup for styling, we should rather use CSS for that